### DISCUSSION TITLE
Apollo Graphql Federation 2 support

### DISCUSSION THREAD

**Discussion by killjoy2013** [graphql-dotnet/graphql-dotnet#3542] [Q&A] [answered]
Is there a time plan for Apollo Graphql Federation 2 support?

Thanks

**Comment by Shane32** [comment]
See:
- https://github.com/graphql-dotnet/graphql-dotnet/pull/3144

No timeline; but we anticipate it will be in GraphQL.NET version 8

**Comment by sungam3r** [comment]
Yeah, no timeline and this task is hanging (at least on me) with a big debt.

**Comment by sungam3r** [comment]
If I had not dive so much in AOT/Open Telemetry stuff, I would have begun working on it instead.

**Comment by Shane32** [comment, marked as answer]
Apollo Graphql Federation 2 is supported in GraphQL.NET 8.x
